03-monitoring-an-app/002-how-to-create-alarms.md

```diff
@@ -0,0 +1,99 @@
+---
+category: cloud-platform
+expires: 2019-07-01
+---
+# Creating your own custom alerts
+
+## Overview
+Alertmanager allows you define your own alert conditions based on [Prometheus expression language](https://prometheus.io/docs/prometheus/latest/querying/basics) expressions. 
+
+The aim of this document is to provide you with the necessary information to create and send application specific alerts to a Slack channel of your choosing.
+
+## Prerequisites
+This guide assumes the following:
+
+* You have [created a namespace for your application]({{ "/01-getting-started/002-env-create" | relative_url }})
+* Deployed at least one pod in your namespace 
+
+## Creating a slack webhook and amend Alertmanager
+This step requires the Cloud Platform team to create a receiver in [Alertmanager](https://github.com/ministryofjustice/cloud-platform-infrastructure/blob/master/terraform/cloud-platform-components/templates/prometheus-operator.yaml.tpl#L115) and a [Slack webhook](https://api.slack.com/incoming-webhooks).
+
+Use the #ask-cloud-platform Slack channel to request a new alert route in Alertmanager. The team will need the following information:
+  
+- namespace name
+- team name
+- application name
+- slack channel
+
+The team will provide you with a `value` that'll need to be defined in the next step. Please copy it to your clipboard. 
```

Comment:
I find this a bit confusing. Should we perhaps just call it the "severity level" or "custom severity level" or similar?

Reply:
A fair point. How about using the original, "Once completed, the Cloud Platform team will supply you with a value for the severity label. All alerts are routed using this label." instead?

Reply:
Changed in 
[15e81c5](/ministryofjustice/cloud-platform-user-docs/pull/114/commits/15e81c508f1a6b321666886927136e91d82f6925)